### DISCUSSION TITLE
Highliting more then one selector

### DISCUSSION THREAD

**Discussion by CodeSippr** [clientIO/joint#2153] [Q&A]
### Introduction

Hi ,

I have an element consisting of the following markup:

`markup = [ { tagName: "line", selector: "body", }, { tagName: "rect", selector: "header", }, { tagName: "text", selector: "label", }, { tagName: "rect", selector: "resize_border_bottom", }, ];`

Currently, I'm highlighting the body selector with:

`joint.highlighters.mask.add(cellView, "body", "highlighter-selector", { attrs: { stroke: "#43a148", "stroke-width": 4 }, });`

Now, I would like to highlight both the line and the body selectors. How is this possible? :)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
What do you mean? The `line` is referenced by the `body` selector. The `body` is the `line`.

**Reply by CodeSippr** [reply]
Wrong Markup: 

markup = [ 
{ tagName: "line", selector: "line", },
{ tagName: "rect", selector: "body", },
{ tagName: "text", selector: "label", },
{ tagName: "rect", selector: "resize_border_bottom", },
];

**Reply by kumilingus** [reply]
1) You can use 2 highlighters.
```js
joint.highlighters.mask.add(cellView, "line", "highlighter-selector1");
joint.highlighters.mask.add(cellView, "body", "highlighter-selector2");
```
2) You can group the `line` and the `body` and highlight the group
```js
markup = [
  {
    tagName: 'g',
    selector: 'bodyGroup',
    children: [
      { tagName: "line", selector: "line", },
      { tagName: "rect", selector: "body" },
    ]
  },
  { tagName: "text", selector: "label", },
  { tagName: "rect", selector: "resize_border_bottom", },
];

joint.highlighters.mask.add(cellView, "bodyGroup", "highlighter-selector", { deep: true });
```
